### ISSUE TITLE
boto_asg = Resolve issue on Create if tags not Given

### ISSUE THREAD

**Issue by colinbjohnson** [saltstack/salt#14604]
### Steps to reproduce:
```root@salt-dev:/srv/salt# salt-call boto_asg.create test test '["us-east-1a", "us-east-1e"]' 0 0 --log-level debug```
### Result:
```
[DEBUG   ] Reading configuration from /etc/salt/minion
[INFO    ] Using cached minion ID from /etc/salt/minion_id: salt-dev
[DEBUG   ] Configuration file path: /etc/salt/minion
[DEBUG   ] Reading configuration from /etc/salt/minion
Passed invalid arguments: 'NoneType' object is not iterable
Traceback (most recent call last):
  File "/srv/salt/salt/cli/caller.py", line 92, in call
    ret['return'] = func(*args, **kwargs)
  File "/srv/salt/salt/modules/boto_asg.py", line 156, in create
    for tag in tags:
TypeError: 'NoneType' object is not iterable
```
### Desired Result:
If no tags are given, no tags are created.

**Comment by ssgward** [comment]
@colinbjohnson - is this still an issue with the latest salt? If not an issue, can this be closed?

**Comment by rallytime** [comment]
This was fixed with #14607. Closing.
